src/Gallery/Components/GalleryFileExtensions.ts

```diff
@@ -0,0 +1,4 @@
+export const GalleryFileExtensions = {
+  image: ['jpg', 'jpeg', 'png', 'webp'],
```

Comment:
I've never uploaded a Webp. It would be interesting to test to check that it works.

Reply:
yes, `webp` works.